tests/modules/programs/eww/basic-configuration.nix

```diff
@@ -0,0 +1,38 @@
+{ config, pkgs, ... }:
+
+{
+  config = {
+    programs.eww = {
+      enable = true;
+      systemd.enable = true;
+      configYuck = ''
```

Comment:
I guess the `configYuck` and `configScss` are mistakes? Maybe need a separate yuck and scss test, since they are mutually exclusive?

Reply:
Nevermind, I was a bit confused. See my other suggestion about having `yuckConfig` and `scssConfig`.